src/ursa/cli/hitl.py

```diff
@@ -431,6 +432,53 @@ def do_models(self, _: str):
         )
 
 
+mcp_app = FastAPI(
+    title="URSA Server",
+    description="Micro-service for hosting URSA to integrate as an MCP tool.",
+    version="0.1.0",
+)
+
+
+class QueryRequest(BaseModel):
+    agent: str = Literal["arxiv", "plan", "execute", "web", "recall", "chat"]
+    query: str = Field(
+        ..., example="Write the first 1000 prime numbers to a text file."
+    )
```

Comment:
```suggestion
class QueryRequest(BaseModel):
    agent: Literal["arxiv", "plan", "execute", "web", "recall", "chat"]
    query: Annotated[
        str,
        Field(examples=["Write the first 1000 prime numbers to a text file."]),
    ]
```

Let me know if this is what you mean:
* agent is a `str` that can be one of "arvix", "plan", etc. The type for this is `Literal["arxiv", "plan", ...]`. So that should not be the default value.
* query is a `str`, and you want to provide examples through `Field`.